sd_hf_serve/app/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+docker logout
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-west-2.amazonaws.com
+docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference-neuronx:1.13.1-neuronx-py310-sdk2.18.0-ubuntu20.04
+dlc_image_id=$(docker images | grep 763104351884 | grep 1.13.1-neuronx-py310-sdk2.15.0-ubuntu20.04 | awk '{print $3}')
```

Comment:
This line also needs to be upgraded. Maybe test the whole script one time to make sure there is no other issue.

Reply:
Exported DLC images config to the build job spec instead of changing hardcoded values in the script.